src/content/docs/guides/json/0-json_intro.mdx

```diff
@@ -72,8 +72,26 @@ This command creates sub-folders for each type of resource. One of these is name
   ```
 
   </TabItem>
+  
```

Comment:
Need to check the functions you are using as they do not exist.

The below compiles and runs when game_data.json exists in the Resource/json folder within your project:

using SplashKitSDK;

class JsonTutorial
{
    static void Main()
    {
        Json gameData;
        gameData = SplashKit.JsonFromFile("game_data.json");
        string gameTitle = SplashKit.JsonReadString(gameData, "gameTitle");
        
        SplashKit.WriteLine("Game Title: " + gameTitle);
        
        SplashKit.FreeJson(gameData);
    }
}